docs/docs/notification-center/react-components.md

```diff
@@ -238,6 +238,7 @@ The `i18n` prop can accept 2 different types of values
           <li><code>zu</code> (Zulu)</li>
           <li><code>zh</code> (Chinese)</li>
           <li><code>sq</code> (Albanian)</li>
+          <li><code>bs</code> (Bosnian)</li>
```

Comment:
This is not alphabetically sorted by the iso language code. Could you please ammend?

Reply:
Also if you can help us moving the albanian code to its proper place, we would appreciate it.